commit 3e1089a1c5e609c2bd982f1bdfebf7e4b7000f3c
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-08T10:10:26+00:00 (3 months ago)
    
    joyent/node-zkstream#9 use var instead of const so we can run in strict mode ((foo bar))
    joyent/node-zkstream#29 unreliable tests assuming after sync() all watchers have fired
    joyent/node-zkstream#24 remove watchers from index once all event handlers are gone
    joyent/node-zkstream#30 createWithEmptyParents API tests call Buffer.from

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
